generate-workload.py

```diff
@@ -34,5 +37,5 @@
     # check if this is a tweet by looking for message text
     if 'text' in item:
         print "ITEM"
-        print item 
+        print item
```

Comment:
Can you please consider removing above two print statements? i used it originally for debugging. Thanks!